### PR TITLE
Added note about incomplete HTML in tutorial 3.

### PR DESCRIPTION
**Pull request by adamchainz**

### CODE REVIEW

**Comment by adamchainz** [comment]
This comes from this question on the forums: https://forum.djangoproject.com/t/are-html-tags-needed-in-templates/806

File: docs/intro/tutorial03.txt [hunk 1]
```diff
@@ -207,6 +207,10 @@ Put the following code in that template:
         <p>No polls are available.</p>
     {% endif %}
 
+Note: for the sake of shortening the tutorial, this is incomplete HTML. You
+should include the normal ``<!DOCTYPE html>``, ``<head>``, and ``<body>`` tags
+in your own templates.
```
Comment (felixxm) [inline docs/intro/tutorial03.txt]: All HTML examples in tutorials are incomplete, I would move this to the top and add an example of valid HTML, e.g.:
```
.. note::

    For the sake of shortening, all templates examples in tutorials are
    incomplete HTML. You should put them in the valid HTML structure, e.g.:

    .. code-block:: html+django

        <!DOCTYPE html>
        <html lang="en">
          <head>
            <title>Tutorial 03</title>
          </head>
          <body>
            # Template example.
          </body>
        </html>
```

What do you think?

Comment (adamchainz) [reply docs/intro/tutorial03.txt]: I'd prefer to keep the tutorial free of a full template - many users doing the tutorial have seen it before.

Perhaps we could link out to: 
https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#Anatomy_of_an_HTML_document

Comment (felixxm) [reply docs/intro/tutorial03.txt]: So maybe without an example:
```
.. note::

    For the sake of shortening, all templates examples in tutorials are
    incomplete HTML. You should put them in the valid HTML structure. 
```

**Comment by adamchainz** [comment]
👍

**Pull request merged**